dashboard/config/routes.rb

```diff
@@ -354,8 +354,8 @@ module OPS
       get 'school-districts/:state', to: 'school_districts#index', defaults: { format: 'json' }
 
       # Routes used by UI test status pages
-      get 'test_logs/:branch/since/:time', to: 'test_logs#get_logs_since', defaults: { format: 'json' }
-      get 'test_logs/:branch/:name', to: 'test_logs#get_log_details', defaults: { format: 'json' }
+      get 'test_logs/*prefix/since/:time', to: 'test_logs#get_logs_since', defaults: { format: 'json' }
+      get 'test_logs/*prefix/:name', to: 'test_logs#get_log_details', defaults: { format: 'json' }
```

Comment:
These are called [wildcard segments](http://guides.rubyonrails.org/routing.html#route-globbing-and-wildcard-segments) and they can capture any number of route parts.